libexec/alluxio-config.sh

```diff
@@ -151,6 +151,12 @@ if [[ -n "${ALLUXIO_LOGSERVER_HOSTNAME}" && -n "${ALLUXIO_LOGSERVER_PORT}" ]]; t
     ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.remote.logger.type=REMOTE_WORKER_LOGGER"
 fi
 
+# FUSE specific parameters that will be shared to all workers based on ALLUXIO_JAVA_OPTS.
+ALLUXIO_FUSE_JAVA_OPTS_DEFAULT=" -server -Xms1G -Xmx1G -XX:MaxDirectMemorySize=4g"
+ALLUXIO_FUSE_JAVA_OPTS_DEFAULT+=" -Djava.library.path=${ALLUXIO_HOME}/lib"
+ALLUXIO_FUSE_JAVA_OPTS_DEFAULT+=" -Dalluxio.logger.type=FUSE_LOGGER"
+ALLUXIO_FUSE_JAVA_OPTS="${ALLUXIO_FUSE_JAVA_OPTS_DEFAULT} ${ALLUXIO_JAVA_OPTS} ${ALLUXIO_FUSE_JAVA_OPTS}"
```

Comment:
order here is tricky, typically JVM respect the rightmost setting when a java opt  is passed multiple times.
so let's go this way: from most general to most specific settings. in this way, when users' setting ALLUXIO_FUSE_JAVA_OPTS in alluxio-env.sh will be respected.

@calvinjia  do you agree here? shall we fix other env?

Reply:
Yes, most JVMs respect the latter option. I think this is okay and its not worth doing some more bash scripting to try and de-duplicate.